change/@ni-nimble-components-e878875a-13bb-474b-b54e-76adb87bc3fa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Will make a separate thread but chatted offline with Molly about having the mental model of:

1. Pressing ctrl key is required to add to your selection (so having a selection and not holding ctrl w/ shift clears the previous selection, this aligns with Windows explorer)
2. Using the checkboxes is a mental model of always using the ctrl key (they are always additive)

Reply:
I've updated the code to have the behavior you described.